templates/index.html

```diff
@@ -187,21 +187,41 @@ <h2 class="feature-title">Free and Open Source</h2>
     <img src="/assets/bevy_logo_dark.svg" class="bevy-logo-header" alt="Bevy logo"/>
     <div class="media-content features-sponsors">
         <h3>A big thanks to Bevy's generous <a href="https://github.com/sponsors/cart/">Sponsors</a> for making our work sustainable:</h3>
-        <h2>Platinum Sponsors</h2>
-            <a href="https://www.embark-studios.com/"><img src="assets/embark.png" class="platinum-sponsor-image"  alt="Embark Studios logo"/></a>
-            <a href="https://legionlabs.com/"><img src="assets/LegionLabs.png" class="platinum-sponsor-image" style="height: 6.3rem" alt="Legion Labs logo"/></a>
-        <h2>Silver Sponsors</h2>
-        <div class="sponsors-section">
-            <a href="https://agileperception.com/"><img src="assets/agile_perception.png" class="silver-sponsor-image" alt="Agile Perception logo"/></a>
+        <div class="sponsors sponsors--platinum">
+            <h2 class="sponsors__title">Platinum Sponsors</h2>
+            <div class="sponsors__content">
+                <a class="sponsors__link" href="https://www.embark-studios.com">
+                    <img class="sponsors__logo" src="assets/embark.png" alt="Embark Studios logo"/>
+                </a>
+                <a class="sponsors__link" href="https://legionlabs.com">
+                    <img class="sponsors__logo" src="assets/LegionLabs.png" alt="Legion Labs logo"/>
```

Comment:
Legion Labs had a manual scale override to make it closer in size to the embark logo. We should either re-add the override or scale the image itself to ensure we aren't giving unfair prominence to Legion Labs.

Reply:
Works for me!

Reply:
I've added a bigger Embark logo, and now it takes the 100px height. The size difference should be less noticeable (it's still not totally balanced though). Anyway, this should be revisited at some time, it's a little bit tricky to keep them balanced, reponsive, while maintaining the aspect ratio and with good spacing between them… 😅 

Oh, btw, I've found high-quality embark logos on the Arc Riders press kit: https://drive.google.com/drive/folders/1u8D3kF0GKKuouXeOf7qU0a6ReNSOv0hd

Reply:
OK, I've also tweaked the LegionLabs logo. So now both of them have 200px height and same "x-height" (lowercase "x" height). **Good enough?** :P

<img width="675" alt="image" src="https://user-images.githubusercontent.com/188612/161827982-09594dfa-befd-473f-85b0-2f9f83c3bf58.png">

<img width="1109" alt="image" src="https://user-images.githubusercontent.com/188612/161828523-1e417d90-279b-492d-88c6-9c953c6a6e6e.png">

Reply:
Good enough!

Reply:
Agreed!